pkg/runctl/run_manager_test.go

```diff
@@ -1097,6 +1097,95 @@ func Test_RunManager_Start_CreatesTektonTaskRun(t *testing.T) {
 	assert.Assert(t, result != nil)
 }
 
+func Test_RunManager_addTektonTaskRunParamsForJenkinsfileRunnerImage(t *testing.T) {
```

Comment:
There should be a test case that checks if the parameters are really _appended_, not replaced.

Reply:
See https://github.com/SAP/stewardci-core/pull/162/commits/bef36c7b76ee556d6c2acf510d6bf1895791486a